MarvelApiClient/HTTPVerb.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  HttpVerb.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente Gomez on 12/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+enum HTTPVerb {
```

Comment:
The name should be HTTPMethod based on the rfc http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html
And if HTTPMethod extend String (HTTPMethod: String)you would have a direct conversion to the raw value.
HTTPMethod.GET.rawValue == "GET"

Also for BothamNetworking we should list all Method available in the standard.
OPTIONS GET HEAD POST PUT DELETE TRACE CONNECT